scripts/help_md_to_yml

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env ruby
```

Comment:
Aside: I hadn't noticed this file before, but based on the implementation, it might need to be updated for the new folder structure? Or removed if we don't have much value for it.

Reply:
We were just talking about that yesterday. It seems that this file is not used anymore.

Reply:
yes we recently discussed, I think its better to just delete this. i dont think this is used anymore